templates/zerver/help/import-data-from-slack.md

```diff
@@ -10,10 +10,13 @@
    `slack_data.zip`. Currently we only support standard export. See
    https://get.slack.help/hc/en-us/articles/201658943-Export-data-and-message-history
    for more detail on this step.
-2. Convert the zip file `slack_data.zip` to Zulip export format using the
-   command `./manage.py convert_slack_data <slack_zip_file> <organization_name> --output <output_dir>`.
-3. Import the converted data. If you are importing into an existing database,
-   run the command `./manage.py import --import-into-nonempty <output_dit>`,
+2. Generate a Slack API token using Slack's test
+   [token generator](https://api.slack.com/custom-integrations/legacy-tokens)
+   to import all of the necessary data.
```

Comment:
Maybe say "to import emails" instead and explain that the current version of slack export file doesn't have emails for some reason, so this workaround is necessary.